math/isautomorphic.go

```diff
@@ -0,0 +1,34 @@
+// isautomorphic.go
+// description: Checks whether a whole number integer is Automorphic or not. If number < 0 then returns false.
+// details:
+// In mathematics, a number n is said to be a Automorphic number if the square of n ends in the same digits as n itself.
+// ref: (https://en.wikipedia.org/wiki/Automorphic_number)
+// time complexity: O(log10(N))
+// space complexity: O(1)
+// author: [SilverDragonOfR](https://github.com/SilverDragonOfR)
+
+package math
+
+// constraints package: to work with any type of signed/unsigned integer
+import (
+	f "github.com/TheAlgorithms/Go/constraints"
```

Comment:
Please don't rename the package here - I don't see `f` to be a meaningful name for this to be used within a package.